crates/fuel-core/src/graphql_api/storage.rs

```diff
@@ -123,6 +123,41 @@ pub enum Column {
     AssetsInfo = 25,
     /// Index of the coins that are available to spend.
     CoinsToSpend = 26,
+    /// See [`DaCompressionTemporalRegistryAddressV2`](da_compression::v2::DaCompressionTemporalRegistryAddressV2)
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalRegistryAddressV2 = 27,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalAddressMerkleData = 28,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalAddressMerkleMetadata = 29,
+    // See [`DaCompressionTemporalRegistryAssetIdV2`](da_compression::v2::DaCompressionTemporalRegistryAssetIdV2)
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalRegistryAssetIdV2 = 30,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalAssetIdMerkleData = 31,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalAssetIdMerkleMetadata = 32,
+    /// See [`DaCompressionTemporalRegistryContractIdV2`](da_compression::v2::DaCompressionTemporalRegistryContractIdV2)
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalRegistryContractIdV2 = 33,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalContractIdMerkleData = 34,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalContractIdMerkleMetadata = 35,
+    /// See [`DaCompressionTemporalRegistryScriptCodeV2`](da_compression::v2::DaCompressionTemporalRegistryScriptCodeV2)
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalRegistryScriptCodeV2 = 36,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalScriptCodeMerkleData = 37,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalScriptCodeMerkleMetadata = 38,
+    /// See [`DaCompressionTemporalRegistryPredicateCodeV2`](da_compression::v2::DaCompressionTemporalRegistryPredicateCodeV2)
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalRegistryPredicateCodeV2 = 39,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalPredicateCodeMerkleData = 40,
+    #[cfg(feature = "fault-proving")]
+    DaCompressionTemporalPredicateCodeMerkleMetadata = 41,
```

Comment:
It would be nice if we could break out all of these to a separate enum to avoid having to repeat the repeated cfg attribute. But that probably needs a bit of other changes to work as well so we can save that for a follow-up.

Reply:
sounds good, will keep in mind